### PR TITLE
JW-1016 cleanup utils

### PR DESCRIPTION
**Pull request by michal-clapinski**

### CODE REVIEW

File: test/atomic-test/atomic-test.vcxproj [hunk 1]
```diff
@@ -38,7 +38,7 @@
     <ProjectGuid>{621A6695-3A8D-4F94-942B-D8745888E768}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>atomic-test</RootNamespace>
-    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
```
Comment (m-kostrzewa) [inline test/atomic-test/atomic-test.vcxproj]: can we keep LatestTargetPlatformVersion?

Comment (m-kostrzewa) [reply test/atomic-test/atomic-test.vcxproj]: (just a reminder I guess)

File: utils/vtest/all_tests_run.ps1 [hunk 2]
```diff
@@ -1,32 +1,27 @@
...
+       [Parameter(Mandatory = $true)] [string] $TestsFolder,
+       [Parameter(Mandatory = $false)] [string] $VtestPath = "vtest.exe",
+       [Parameter(Mandatory = $false)] [string] $ExtensionName = "vRouter forwarding extension")
 
 function RestartExtension {
-    Disable-VMSwitchExtension -Name $ExtensionName -VMSwitchName $VMSwitchName -ErrorVariable err | out-null
-    If ($err) {
-        Write-Host "Error while disabling the extension"
-        exit 1
+    Disable-VMSwitchExtension -Name $ExtensionName -VMSwitchName $VMSwitchName -ErrorVariable Err | out-null
```
Comment (jacek-izykowski) [inline utils/vtest/all_tests_run.ps1]: It's "Out-Null".

File: utils/vtest/all_tests_run.ps1 [hunk 3]
```diff
@@ -1,32 +1,27 @@
...
+    If ($Err) {
+        throw "Error while disabling the extension"
     }
-    Enable-VMSwitchExtension -Name $ExtensionName -VMSwitchName $VMSwitchName -ErrorVariable err | out-null
-    If ($err) {
-        Write-Host "Error while enabling the extension"
-        exit 2
+    Enable-VMSwitchExtension -Name $ExtensionName -VMSwitchName $VMSwitchName -ErrorVariable Err | out-null
```
Comment (jacek-izykowski) [inline utils/vtest/all_tests_run.ps1]: It's "Out-Null".

**Pull request merged**